<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JaqubCreateConnectionDialog</class>
 <widget class="QDialog" name="JaqubCreateConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>524</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Connection</string>
  </property>
  <property name="windowIcon">
   <iconset theme="internet-chat">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="icon">
     <iconset theme="settings">
      <normaloff>.</normaloff>.</iconset>
    </attribute>
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="connectionNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection Name</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="KTextEdit" name="connectionNameField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="connectionTab">
    <attribute name="icon">
     <iconset theme="internet-chat">
      <normaloff>.</normaloff>.</iconset>
    </attribute>
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="KHistoryComboBox" name="connectionTypeCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="historyItems">
      <stringlist/>
     </property>
    </widget>
    <widget class="KLineEdit" name="connectionUriField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNoEditMenu</set>
     </property>
     <property name="placeholderText">
      <string>Connection URI</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="connectionTab2">
    <attribute name="icon">
     <iconset theme="internet-chat">
      <normaloff>.</normaloff>.</iconset>
    </attribute>
    <attribute name="title">
     <string>Connection 2</string>
    </attribute>
    <widget class="KLineEdit" name="usernameField">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>512</number>
     </property>
     <property name="contextMenuEnabled">
      <bool>false</bool>
     </property>
     <property name="urlDropsEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="KLineEdit" name="passwordField">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>512</number>
     </property>
     <property name="contextMenuEnabled">
      <bool>false</bool>
     </property>
     <property name="urlDropsEnabled">
      <bool>false</bool>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
     <property name="passwordMode">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="KTitleWidget" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Connection</string>
   </property>
   <property name="comment">
    <string>Enter in the information needed to connect to others.</string>
   </property>
   <property name="icon">
    <iconset theme="internet-chat">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoHideTimeout">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>JaqubCreateConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>JaqubCreateConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
